Tiniest NiT on CE ? (#320)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

acknowledgement when there is a transition from non-CE-marked to CE-marked.

If the Ack-Eliciting Threshold is 0, every ack-eliciting packet is immediately

> acknowledged (both when the packet was CE marked and when not). If the Ack-Eliciting Threshold

Reviewer: Is this better?

```suggestion
acknowledged, whether it is CE marked or not. If the Ack-Eliciting Threshold
```
